bindings/node/lib/clientEncryption.js

```diff
@@ -267,79 +327,239 @@ module.exports = function (modules) {
       const stateMachine = new StateMachine({
         bson,
         proxyOptions: this._proxyOptions,
-        tlsOptions: this._tlsOptions,
-        session: options.session
```

Comment:
I assume the idea is to add back support for this for all APIs in one go at some point?

Reply:
Yup, we have a drivers ticket for it - https://jira.mongodb.org/browse/DRIVERS-2389

Hopefully it gets picked up soon (maybe I'll talk to Jeff about doing it, it sounds relatively straightforward and interesting).